### PR TITLE
BZ#2059725 Added oc edit storageclass command

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.8+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2059725

Link to docs preview:
https://54092--docspreview.netlify.app/openshift-enterprise/latest/storage/expanding-persistent-volumes.html#add-volume-expansion_expanding-persistent-volumes

QE review:
- [X] QE has approved this change.

### CODE REVIEW

**Comment by cbippley** [comment]
@duanwei33 Could you take a look at this? Thanks!

**Comment by cbippley** [comment]
@duanwei33 Could you take a look at this? Thanks!

File: modules/storage-expanding-add-volume-expansion.adoc [hunk 1]
```diff
@@ -12,7 +12,16 @@ have the `allowVolumeExpansion` field set to `true`.
 
 .Procedure
 
-* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute.
+* Edit the `StorageClass` object and add the `allowVolumeExpansion` attribute by using the following command:
++
+[source,terminal]
+----
+$ oc edit storageclass <storage_class_name> \// <1> 
+   -n <storage_class_namespace> <2>
```
Comment (duanwei33) [inline modules/storage-expanding-add-volume-expansion.adoc:20]: Hi @cbippley Storageclass is a cluster-wide resource, so no need the "-n <storage_class_namespace>"

File: modules/storage-expanding-add-volume-expansion.adoc [hunk 2]
```diff
@@ -12,7 +12,16 @@ have the `allowVolumeExpansion` field set to `true`.
...
+----
+<1> Specifies the name of the storage class.
+<2> Specifies the namespace that contains the storage class.
```
Comment (duanwei33) [inline modules/storage-expanding-add-volume-expansion.adoc:23]: This should be removed accordingly.

**Comment by cbippley** [comment]
Thank you @duanwei33 How does it look now?

**Comment by duanwei33** [comment]
Thanks for the update, it looks good to me.

/lgtm

**Comment by stevsmit** [comment]
@cbippley 1 nit. Looks good overall. Feel free to make the change and then add the merge review label. :D

**Comment by cbippley** [comment]
@stevsmit Can you tell me the change I should make? Thank you!

**Pull request merged**